src/lib/FlightTasks/tasks/FlightTask/FlightTask.hpp

```diff
@@ -166,7 +166,7 @@ class FlightTask : public ModuleParams
 	/**
 	 * Call this whenever a parameter update notification is received (parameter_update uORB message)
 	 */
-	void handleParameterUpdate()
+	void virtual handleParameterUpdate()
```

Comment:
I hope it's ok to stick to the convention of putting virtual at the beginning.

Reply:
Done in https://github.com/PX4/Firmware/pull/12751/commits/2166b2f877185733fade41ec7906ae777b03ef6b